source/common/http/utility.h

```diff
@@ -391,6 +406,17 @@ struct LocalReplyData {
 void sendLocalReply(const bool& is_reset, const EncodeFunctions& encode_functions,
                     const LocalReplyData& local_reply_data);
 
+/**
+ * Prepares a locally generate response modifying headers and rewriting the
+ * response.
+ */
```

Comment:
Maybe just `Prepares a locally generated response` /w parameter docs? As is it is a bit hard to grok